Sources/SwiftTUI/Views/Controls/Button.swift

```diff
@@ -1,60 +1,69 @@
 import Foundation
 
-public struct Button: View, PrimitiveView {
-    public let n: String
+public struct Button<Label>: View, PrimitiveView where Label : View {
+    public let label: Label
     public let action: () -> Void
 
-    public init(_ text: String, action: @escaping () -> Void) {
-        self.n = text
+    public init(action: @escaping () -> Void, @ViewBuilder label: () -> Label) {
+        self.label = label()
         self.action = action
     }
 
     static var size: Int? { 1 }
 
     func buildNode(_ node: Node) {
-        node.control = ButtonControl(text: n, action: action)
+        node.addNode(at: 0, Node(view: label.view))
+        node.control = ButtonControl(action: action)
+        for i in 0 ..< node.children[0].size {
```

Comment:
The behavior of the SwiftUI Button, used with a label with multiple views, is that it uses an implicit VStack. So here instead of having to worry about multiple subviews, you can add a child node for a VStack with the label inside, and only add that VStack's control. Take a look at `ScrollView`.